test_files/export_multi/export_multi.ts

```diff
@@ -15,7 +15,15 @@ export const {
   ORANGE,
 } = Fruit;
 
-export const {a, b} = {
+export const {a, b, c, d, e, f, g, h, i, j} = {
```

Comment:
can you add a comment why we're testing 10 elements here?

Reply:
Added

Reply:
Added